MANIFEST.in

```diff
@@ -5,6 +5,4 @@ include LICENSE
 include README.rst
 include requirements.txt
 recursive-exclude exercise_perseus_render/* *pyc
-recursive-include exercise_perseus_render/static *.*
-recursive-include exercise_perseus_render/build *.json
-recursive-exclude exercise_perseus_render/assets *
+recursive-include exercise_perseus_render *
```

Comment:
from your experimenting, have you found that we definitely need everything in here? Or is there a chance that in the future we might exclude additional files?